### ISSUE TITLE
Missing runtime in Nuget - Whisper.net.Runtime.Cuda

### ISSUE THREAD

**Issue by kitsim93** [sandrohanea/whisper.net#214]
Unable to find package Whisper.net.Runtime.Cuda. No packages exist with this id in source(s).
I can't find any Cuda runtime for 1.7.0 in Nuget.

[https://www.nuget.org/packages?q=whisper.net]

**Comment by sandrohanea** [comment]
Indeed, I had some problems to upload it to nuget as the package is quite big (300MB) and the limit on nuget is 250.
I split it into 2:
 - [Whisper.net.Runtime.Cuda.Windows](https://www.nuget.org/packages/Whisper.net.Runtime.Cuda.Windows/)
 - [Whisper.net.Runtime.Cuda.Linux](https://www.nuget.org/packages/Whisper.net.Runtime.Cuda.Linux/)
 
 And the "Big" package is now just an empty package with 2 depenencies on these 2.
 
 It is enough to install [Whisper.net.Runtime.Cuda](https://www.nuget.org/packages/Whisper.net.Runtime.Cuda) and the Windows and Linux packages will be installed as transitive dependencies. 
 
 However, if you want to target a single OS, you can now get a dependency on the specific OS package.
